common/meta/types/proto/meta.proto

```diff
@@ -48,6 +48,56 @@ message ExportedChunk {
   repeated string data = 10;
 }
 
+message WatchRequest {
+  // key is the key to register for watching.
+  string key = 1;
+
+  // key_end is the end of the range [key, key_end] to watch.
+  // if key_end is None, then watch only key.
+  optional string key_end = 2;
+
+  enum FilterType {
+    // recv all kind update event.
+    ALL = 0;    
+    // filter only update event.
+    UPDATE = 1;
+    // filter only delete event.
+    DELETE = 2;
+  }  
+  FilterType filter_type = 3;
+}
+
+message Event {
+  enum EventType {
+    Update = 0;
+    Delete = 1;
+  }
+
+  string key = 1;
+
+  EventType event = 2;
+
+  // current hold current value of key(if any)
+  optional bytes current = 3;
+
+  // prev value of key
+  optional bytes prev = 4;
+}
+
+message WatchResponse {
+  //ResponseHeader header = 1;
+  // watch_id is the ID of the watcher that corresponds to the response.
+  int64 watch_id = 1;
```

Comment:
Is this field still in use?

Reply:
> Is this field still in use?

i think client still need a id to identify watcher.

Reply:
> i think client still need a id to identify watcher.

And what does a client need a watch id for?...